### PR TITLE
board-image/revyos-sipeed-lpi4a: Bump to 0.20240720.0

### PR DESCRIPTION
**Pull request by wychlw**
Bump revyos-sipeed-lpi4a from 0.20231210.0 to 0.20240720.0-matrix.bot.

Identifier: [HASH[aa9a7a87f6e08bd49a81dce55770765cf0fcdb5569864d91116af147]]

This PR is made by ruyi-index-updator bot.

### CODE REVIEW

File: manifests/board-image/0.20240720.0-matrix.bot.toml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+format = "v1"
+[[distfiles]]
+name = "root-lpi4a-20240720_171951.ext4.zst"
```
Comment (xen0n) [inline manifests/board-image/0.20240720.0-matrix.bot.toml:3]: What's this file?

Comment (wychlw) [reply manifests/board-image/0.20240720.0-matrix.bot.toml]: Sorry due to repo cache in local, the bot uploads some unexpected file. It should been fixed.

**Comment by xen0n** [comment]
It's a little bit strange that the version numbers contain "matrix.bot" while in fact the respective packages are not generated by the bot. Could you remove the marker or at least move it to a comment *inside* the file? After that it should be ready to go. Thanks!

**Comment by wychlw** [comment]
> It's a little bit strange that the version numbers contain "matrix.bot" while in fact the respective packages are not generated by the bot. Could you remove the marker or at least move it to a comment _inside_ the file? After that it should be ready to go. Thanks!

This part in the version number is because, the pr and file is generated by a CI in support matrix. So I made it as an identifier. If it is unnecessary, I'll move it in the file, cause if the file has any problems, an identifier is still needed to indicate something is happening in the CI program instead of a human error.

**Pull request merged**